common/unsafe/src/test/java/org/apache/spark/unsafe/types/UTF8StringSuite.java

```diff
@@ -730,4 +726,49 @@ public void testToLong() throws IOException {
       assertFalse(negativeInput, UTF8String.fromString(negativeInput).toLong(wrapper));
     }
   }
+  @Test
+  public void trimsChar() {
```

Comment:
Could you split this test case into three test cases for trim, trimLeft, trimRight?